darts/models/smoothing_model.py

```diff
@@ -0,0 +1,109 @@
+"""
+Smoothing Model Base Class
+------------------------------
+
+A smoothing model more accurate value from measured values of a time series
+Using the current value and historic values as follows:
+
+.. math:: y_{t+1} = f(y_{t+1}, y_t, y_{t-1}, ..., y_1),
+
+where :math:`y_t` represents the time series' smoothed value(s) at time :math:`t`.
+
+The main functions are `fit()` and `predict()`. `fit()` learns the function `f()`, over the history of
+one or several time series. The function `predict()` applies `f()` on one or several time series in order
+to obtain more accurate predictions of the measured values(s)
+"""
+
+from typing import Optional
+from itertools import product
+from abc import ABC, abstractmethod
+from inspect import signature
+import numpy as np
+import pandas as pd
+
+from ..timeseries import TimeSeries
+from ..logging import get_logger, raise_log, raise_if_not
+
+logger = get_logger(__name__)
+
+
+class SmoothingModel(ABC):
+
+    """ The base class for smoothing models. It defines the *minimal* behavior that all smoothing models have to support.
+        The signatures in this base class are for "local" models handling only one series and no covariates.
+        Sub-classes can handle more complex cases.
+    """
+    @abstractmethod
+    def __init__(self):
+        # The series used for training the model through the `fit()` function.
+        # This is only used if the model has been fit on one time series.
```

Comment:
Does this sentence still apply? Is there the possibility to fit some filtering models on several series?

Reply:
not anylonger. Thanks for the hint.